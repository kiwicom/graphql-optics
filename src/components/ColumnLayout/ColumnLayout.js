import React from 'react'

const style = {
  display: 'flex',
  flexDirection: 'row',
}

export default ({ children }) => <div style={style}>{children}</div>
