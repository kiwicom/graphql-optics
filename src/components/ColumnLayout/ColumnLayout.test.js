import React from 'react'
import ColumnLayout from './ColumnLayout'
import renderer from 'react-test-renderer'

it('renders children', () => {
  const children = <div>Hello!</div>
  const tree = renderer.create(<ColumnLayout>{children}</ColumnLayout>).toJSON()
  expect(tree).toMatchSnapshot()
})
