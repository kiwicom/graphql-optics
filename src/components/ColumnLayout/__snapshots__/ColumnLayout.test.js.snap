// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders children 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
    }
  }
>
  <div>
    Hello!
  </div>
</div>
`;
